config/environments/development.rb

```diff
@@ -120,5 +120,7 @@
 
   config.sequencescape_url = 'http://localhost:6630'
 
+  config.aker_email = 'aker@sanger.ac.uk'
```

Comment:
I'm not sure you need to specify `aker_email` separately in each environment. I think it'd be simpler if it was the same in all environments, so I'd put this in `application.rb` instead.

Reply:
Ok, because that would require changes in the deployment project for the moment I'll leave it in the env files config for the moment